lib/cli/interactive-setup/service.js

```diff
@@ -130,14 +145,34 @@ module.exports = {
         projectDir,
         projectName,
       });
-    } else {
-      const projectType = await projectTypeChoice();
-      if (projectType === 'other') {
+    } else if (context.options['template-url']) {
+      projectName = await resolveProjectNameInput(context.options, workingDir);
+      projectDir = join(workingDir, projectName);
+      const templateUrl = context.options['template-url'];
+      try {
         process.stdout.write(
-          '\nRun “serverless create --help” to view available templates and create a new project ' +
-            'from one of those templates.\n'
+          `\n${chalk.green(`Downloading template from provided url: ${templateUrl}...`)}\n`
+        );
+        await downloadTemplateFromRepo(templateUrl, null, projectName, { silent: true });
+      } catch (err) {
+        throw new ServerlessError(
```

Comment:
I would probably add `If (err.constructor.name !== 'SeverlessError') throw error` to not accidentally silence eventual programmer error